eslint.config.js

```diff
@@ -82,11 +82,6 @@ export default [
       "no-implicit-coercion": "error",
       "no-inner-declarations": "error",
       "no-lonely-if": "error",
-      "no-restricted-syntax": [
-        "error",
-        // `!foo === bar` and `!foo !== bar`
-        'BinaryExpression[operator=/^[!=]==$/] > UnaryExpression.left[operator="!"]',
-      ],
```

Comment:
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-negation-in-equality-check.md